go.sum

```diff
@@ -1,31 +1,498 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Lots of dependencies being added here that I didn't expect from removing pointers.

Reply:
good catch maybe the go.mod go.sum files were updated by mistake? running `go mod tidy` and rechecking in the `go.mod go.sum` files should resolve this.